drgn/helpers/linux/slab.py

```diff
@@ -244,3 +252,70 @@ def _slab_page_objects(page: Object, slab: Object) -> Iterator[Object]:
         slab = cast(slab_type, page)
         if slab.slab_cache == slab_cache:
             yield from _slab_page_objects(page, slab)
+
+
+@overload
+def find_containing_slab_cache(addr: Object) -> Object:
+    """
+    Get the slab cache that an address (given as an :class:`.Object`) was
+    allocated from, if any.
+    :param addr: Address to look up.
```

Comment:
Need a newline here to make Sphinx happy.

```suggestion
    allocated from, if any.

    :param addr: Address to look up.
```

Reply:
Space added!